### DISCUSSION TITLE
How to use extension?

### DISCUSSION THREAD

**Discussion by scarlion1** [lencx/Noi#62] [Q&A] [answered]
I'm using the Linux client and would like to use the `export-chatgpt` extension but not clear how.  The files from https://github.com/lencx/Noi/tree/d29a2e1368d20f868c32dea457fe27784a2bbed4/extensions/noi-export-chatgpt are in `~/.config/Noi/extensions` and I've opened the Settings > Extensions and enabled the extension.  Now what?

**Comment by fraschm1998** [comment, marked as answer]
yeah the documentation can definitely use some work, however I just figured it out. There seems to be a bug sometimes where the bottom bar wont show up (on linux at least using hyprland, solved by going full screen then exiting full screen) so you can't actually see the icons for the extensions. Essentially what you need to do is enable the desired extensions then click on the icon affiliated with them on the bottom bar which will change the UI to look similar to the following:

https://noi.nofwl.com/readme/noi-theme-dark.jpg

**Reply by scarlion1** [reply]
Thanks!  Going full screen did the trick and was able to export a PDF of a chat.
